### PR TITLE
Prac 09 feedback

### PR DESCRIPTION
**Pull request by qbouet**
Hi @Quinn-Stocker

Could you please review my Prac 09?

Thank you
Kind Regards,
Quentin

### CODE REVIEW

File: prac_09/band.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    def __repr__(self):
+        """Return a string representation of a Band."""
+        return str(self)
+
+    def add(self, musician):
+        """Add a musician to band"""
+        self.musicians.append(musician)
+
+    def play(self):
+        """Play band."""
```
Comment (Quinn-Stocker) [inline prac_09/band.py]: Possibly add a more descriptive Docstring, otherwise good

File: prac_09/car.py
Comment (Quinn-Stocker) [inline prac_09/car.py:1]: Code seems good

File: prac_09/my_band.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
+"""Band example with list of musicians."""
+from band import Band
+from musician import Musician
+from guitar import Guitar
+
+
+def main():
+    band = Band("Extreme")
```
Comment (Quinn-Stocker) [inline prac_09/my_band.py:9]: Might need a Docstring here to describe the process

File: prac_09/silver_service_taxi.py
Comment (Quinn-Stocker) [inline prac_09/silver_service_taxi.py:1]: Code seems good

File: prac_09/silver_service_taxi_test.py
Comment (Quinn-Stocker) [inline prac_09/silver_service_taxi_test.py:1]: Code seems good

File: prac_09/taxi.py
Comment (Quinn-Stocker) [inline prac_09/taxi.py:1]: Code seems good

File: prac_09/taxi_simulator.py [hunk 3]
```diff
@@ -0,0 +1,60 @@
+"""
+CP1404/CP5632 Practical
+Taxi Simulator
+"""
+
+from taxi import Taxi
+from silver_service_taxi import SilverServiceTaxi
+
+MENU = """q)uit, c)hoose taxi, d)rive"""
```
Comment (Quinn-Stocker) [inline prac_09/taxi_simulator.py:9]: possibly add a docstring here to state that this is where you defined variables

File: prac_09/taxi_simulator.py [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+
+def main():
+    """Taxi Simulator"""
```
Comment (Quinn-Stocker) [inline prac_09/taxi_simulator.py:13]: Docstring to state this is the definition of variables

File: prac_09/taxi_simulator.py [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+    taxis = [Taxi("Prius", 100), SilverServiceTaxi("Limo", 100, 2), SilverServiceTaxi("Hummer", 200, 4)]
+    total_bill = 0
+    current_taxi = None
+    print("Let's drive!")
+    print(MENU)
+    choice = input(">>> ").lower()
+    while choice != "q":
```
Comment (Quinn-Stocker) [inline prac_09/taxi_simulator.py:23]: Possibly add a Docstring here to state what you are doing

File: prac_09/taxi_test.py
Comment (Quinn-Stocker) [inline prac_09/taxi_test.py:1]: All good, just an idea maybe add a docstring somewhere to state what the desired output should be so you can ensure that the test was successful

File: prac_09/unreliable_car.py
Comment (Quinn-Stocker) [inline prac_09/unreliable_car.py:1]: Code seems good

File: prac_09/unreliable_car_test.py
Comment (Quinn-Stocker) [inline prac_09/unreliable_car_test.py:1]: Code is good

**Review by Quinn-Stocker — COMMENTED** [review]
Code seems Good just some docstrings might be needed

File: prac_09/band.py (hunk 1 above)
Comment (qbouet) [reply prac_09/band.py]: will do, thank you

File: prac_09/my_band.py (hunk 2 above)
Comment (qbouet) [reply prac_09/my_band.py:9]: ok

**Comment by qbouet** [comment]
Thanks @Quinn-Stocker, I appreciated the review

**Pull request merged**